src/components/Drawer.js

```diff
@@ -0,0 +1,195 @@
+import { Grid, MenuItem } from '@material-ui/core';
+import { PersonRounded } from '@material-ui/icons';
+import { makeStyles, useTheme, withStyles } from '@material-ui/core/styles';
+import AppBar from '@material-ui/core/AppBar';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
```

Comment:
I think tree shaking is enabled in default webpack bundle. So, no need to export each component separately ..